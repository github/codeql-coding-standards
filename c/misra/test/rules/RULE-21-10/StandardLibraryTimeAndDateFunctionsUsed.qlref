rules/RULE-21-10/StandardLibraryTimeAndDateFunctionsUsed.ql