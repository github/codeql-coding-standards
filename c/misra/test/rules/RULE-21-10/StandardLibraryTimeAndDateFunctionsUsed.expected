| test.c:6:18:6:21 | call to time | Call to banned function $@. | time.h:53:8:53:11 | time | time |
| test.c:9:19:9:23 | call to ctime | Call to banned function $@. | time.h:60:7:60:11 | ctime | ctime |
| test.c:17:3:17:6 | call to time | Call to banned function $@. | time.h:53:8:53:11 | time | time |
| test.c:18:8:18:16 | call to localtime | Call to banned function $@. | time.h:58:12:58:20 | localtime | localtime |
| test.c:19:3:19:10 | call to wcsftime | Call to banned function $@. | wchar.h:139:8:139:15 | wcsftime | wcsftime |
